dnf-docker-test/features/list-repo-installed.feature

```diff
@@ -0,0 +1,43 @@
+Feature: DNF/Behave test list installed packages from repository
+
+    @setup
+    Scenario: Feature Setup
+        Given repository "test" with packages
+           | Package | Tag      | Value  |
+           | TestA   | Requires | TestB  |
+           | TestB   | Version  | 4.2    |
+          And repository "test2" with packages
+           | Package | Tag      | Value  |
+           | TestC   | Requires | TestD  |
+           | TestD   | Arch     | x86_64 |
+         When I save rpmdb
+          And I enable repository "test"
+          And I enable repository "test2"
+          And I successfully run "dnf install -y TestA TestC"
+         Then rpmdb changes are
+           | State     | Packages                   |
+           | installed | TestA, TestB, TestC, TestD |
+
+    Scenario: All repos enabled
+         When I successfully run "dnf list installed"
+         Then the command stdout should match regexp "TestA.*\nTestB.*4\.2.*\nTestC.*\nTestD\.x86_64"
+
+    Scenario: Installed from repo "test"
+         When I successfully run "dnf repository-packages -q test list installed"
```

Comment:
The documentation states:
`dnf [options] repository-packages <repoid> list --installed [<package-name-spec>...]`
Please add test according to documentation (with --install option)